doc/sphinx/conf.py

```diff
@@ -183,6 +183,7 @@ def setup(app):
 nitpicky = True
 
 nitpick_ignore = [ ('token', token) for token in [
+    'assums',
     'binders',
     'collection',
     'command',
```

Comment:
Now command is defined here: https://coq.github.io/doc/master/refman/language/gallina-specification-language.html#grammar-token-command
So I think this could be removed.